content/basics/use-ably.textile

```diff
@@ -0,0 +1,119 @@
+---
+title: How to use Ably
+meta_description: "A brief introduction on the best options available for using Ably"
+---
+
+This topic will briefly cover some of the APIs, SDKs, and adapters that Ably provides.
```

Comment:
Rewrite. Opening paragraph should frame the document not just summarize. An example of what this might look like is, "Ably provides both REST and realtime client libraries. Determining whether to use the REST or realtime libraries is essential in the architecture of your applications."